### ISSUE TITLE
F# IDE hangs with beachball when running FindExtensionMethods on FSharpMemberOrFunctionOrValue

### ISSUE THREAD

**Issue by nosami** [mono/monodevelop#5118]
Looking at the log file I can see that GetExtensionMethods is running on the UI thread.

```
DEBUG [2018-06-20 13:35:44Z]: [UI - true] LanguageService: GetExtensionMethods: proj:/Users/jason/src/monodevelop/main/external/fsharpbinding/MonoDevelop.FSharpBinding/MonoDevelop.FSharp.fastbuild.fsproj, file:/Users/jason/src/monodevelop/main/external/fsharpbinding/MonoDevelop.FSharpBinding/RefactoringOperationsHandler.fs, symbol:FSharpMemberOrFunctionOrValue
```

> VS bug [#636110](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/636110)

**Comment by nosami** [comment]
![getextensionmethodshang](https://user-images.githubusercontent.com/667194/41660774-eb9adaa6-7494-11e8-9c38-aea9738e8e72.gif)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-15] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #5128 merged by: Jason Imison  
 Author: mono  
 Commit: c9659a701e65e264960b4fd8f815eb1ad1f5f075 (xamarin/monodevelop)
